src/tools/pipelines.ts

```diff
@@ -0,0 +1,201 @@
+import { z } from 'zod';
+
+import type { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
+import { handleCliOutput } from '../utils/handle-cli-output.js';
+import { CommandBuilder } from '../utils/command-builder.js';
+import { TOOL_COMMAND_MAP } from '../utils/tool-commands.js';
+import type { HerokuREPL } from '../repl/heroku-cli-repl.js';
+import type { McpToolResponse } from '../utils/mcp-tool-response.js';
+
+/**
+ * Schema for creating a new pipeline.
+ */
+export const pipelinesCreateOptionsSchema = z.object({
+  name: z.string().describe('Name of the pipeline to create'),
+  stage: z.string().optional().describe('Stage of first app in pipeline (e.g., production, staging, development)'),
```

Comment:
I will suggest the following change:

```suggestion
  stage: z
    enum(['development', 'staging', 'production'])
    .describe('Stage of first app in pipeline (e.g., production, staging, development)'),
```

That is: making clear the only allowed values on the JSON schema and making it required, because if the CLI command cannot infer the stage it will enter into interactive mode and timeout.